src/main/java/it/gov/pagopa/receipt/pdf/datastore/BizEventToReceipt.java

```diff
@@ -106,7 +106,7 @@ public void processBizEventToReceipt(
                 discarder++;
 
                 //Error info
-                msg = String.format("Error to process event with id %s", bizEvent.getId());
+                msg = String.format("[BizEventToReceiptProcessor] Error to process event with id %s", bizEvent.getId());
```

Comment:
ditto and add trace on discarder biz event too.
Could be useful for troubleshooting